### PR TITLE
[bitnami/postgresql] extended configuration for read PVC

### LINKED ISSUE #11449: [postgresql] readReplicas.persistence.existingClaim
### Name and Version

bitnami/postgresql 11.6.19

### What is the problem this feature will solve?

Reinstalling the PostgreSQL cluster required a bit of cleanup. In particular PVC for replica needs to be removed first. This is not a problem for the master as `primary.persistence.existingClaim` configuration exists.

### What is the feature you are proposing to solve the problem?

Introduce `readReplicas.persistence.existingClaim` to be able to specify PVC.

### What alternatives have you considered?

_No response_

### PR DESCRIPTION
**Pull request by alexey-anufriev**
### Description of the change

Introduced additional configuration to specify read replica PVC.

### Benefits

While re-installing the chart it is not necessary anymore to do a cleanup of PVC, those may be just reused.

### Possible drawbacks

All the read replicate will be using the same PVC.

### Applicable issues

  - fixes #11449

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by carrodher — DISMISSED** [review]
Thanks for this PR, changes LGTM

**Comment by carrodher** [comment]
In order to fix the failing action, could you please rebase from master?

**Comment by alexey-anufriev** [comment]
@carrodher, sorry, but I tried, git says it is up-to-date, so not sure what I need to do as I do not see the error

**Comment by carrodher** [comment]
You need to fetch the changes from this repo to your fork, see https://levelup.gitconnected.com/how-to-update-fork-repo-from-original-repo-b853387dd471. Then you should be able to do a rebase from the recently updated `master` branch in your fork to the branch used to develop those changes (`11449`)

**Comment by alexey-anufriev** [comment]
thank you @carrodher, I completely forgot that I have a fork, now it should be updated.

**Comment by alexey-anufriev** [comment]
But it seems there are still some problems. Could you please share the details so I can fix it?

**Comment by alexey-anufriev** [comment]
And now it looks green again.

File: bitnami/postgresql/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ name: postgresql
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/postgresql
   - https://www.postgresql.org/
-version: 11.6.21
+version: 11.6.22
```
Comment (carrodher) [inline bitnami/postgresql/Chart.yaml:29]: There is a new version (https://github.com/bitnami/charts/pull/11548) so we need to bump it again
```suggestion
version: 11.6.23
```

**Pull request merged**